<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Игра 'Пятнадцать'</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="c1_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1_8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1_9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1_10">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1_12">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1_13">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1_14">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1_15">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="log">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>351</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="select">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>Подтвердить выбор</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>Новая игра</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Правила игры &amp;quot;Пятнадцать&amp;quot;: &lt;/p&gt;&lt;p&gt;Можно брать 1, 2 или 3 &amp;quot;галочки&amp;quot;.&lt;/p&gt;&lt;p&gt;Начинаете всегда Вы. &lt;/p&gt;&lt;p&gt;У кого останется последняя &amp;quot;галочка&amp;quot; - тот проиграл&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
